example/lib/nocodes_view.dart

```diff
@@ -78,13 +78,13 @@ class _NoCodesViewState extends State<NoCodesView> {
       // Set presentation style config before showing screen
       final config = NoCodesPresentationConfig(
         animated: true,
-        presentationStyle: NoCodesPresentationStyle.push,
+        presentationStyle: NoCodesPresentationStyle.fullScreen,
       );
 
-      await NoCodes.getSharedInstance().setScreenPresentationConfig(config, contextKey: 'kamo_test');
+      await NoCodes.getSharedInstance().setScreenPresentationConfig(config, contextKey: 'your_context_key');
```

Comment:
Let's move to a local constant. It's used in two places, and also it would be easy to find and change each time for testing.